scripts/AsmJsonImportTest.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```

Comment:
There's  a ton of duplication between this and `ASTImportTest.sh`. Maybe we could extract common parts?